examples/declaration_dir/tsconfig.json

```diff
@@ -2,6 +2,8 @@
     "compilerOptions": {
         "sourceMap": true,
         "declaration": true,
-        "declarationMap": true
-    }
+        "declarationMap": true,
+        "declarationDir": "out/types"
+    },
+    "exclude": []
```

Comment:
Lets add comments above this referencing the issue?

See the comments that were removed in https://github.com/aspect-build/rules_ts/commit/28ea0571a2f32616739cb506398d78ab08d9dde8